app/src/main/java/ml/docilealligator/infinityforreddit/services/DownloadMediaService.java

```diff
@@ -272,7 +272,7 @@ private Uri writeResponseBodyToDisk(ResponseBody body, boolean isDefaultDestinat
                     contentValues.put(MediaStore.MediaColumns.DISPLAY_NAME, destinationFileName);
                     contentValues.put(MediaStore.MediaColumns.MIME_TYPE, mediaType == EXTRA_MEDIA_TYPE_VIDEO ? "video/*" : "image/*");
                     contentValues.put(MediaStore.MediaColumns.RELATIVE_PATH, destinationFileUriString);
-                    contentValues.put(mediaType == EXTRA_MEDIA_TYPE_VIDEO ? MediaStore.Video.Media.IS_PENDING : MediaStore.Images.Media.IS_PENDING, 1);
+                    contentValues.put(MediaStore.Video.Media.IS_PENDING, 1);
```

Comment:
This and below should be `MediaStore.MediaColumns.IS_PENDING` to avoid mentioning video or images

Reply:
Done